_inc/search/src/index.jsx

```diff
@@ -0,0 +1,30 @@
+/** @jsx h */
+
+/**
+ * External dependencies
+ */
+import { h, render } from 'preact';
+
+/**
+ * Internal dependencies
+ */
+import JetpackSearchAPI from '../components/api';
+import SearchResults from '../components/search-results';
+import SearchWidget from '../components/search-widget';
+
+const api = new JetpackSearchAPI();
+
+const injectSearchWidget = ( initialValue, target, grabFocus ) => {
+	render(
+		<SearchWidget
+			initialValue={ initialValue }
+			grabFocus={ grabFocus }
+			SearchResults={ SearchResults }
+			api={ api }
```

Comment:
Curious why `api` and `SearchResults` are passed down here instead of importing them directly in `SearchWidget`?

Reply:
@notnownikki: Shall I go ahead and move the `api` and `SearchResults` imports into the `SearchWidget` component?